Lombiq.NodeJs.Extensions.Samples/Lombiq.NodeJs.Extensions.Samples.csproj

```diff
@@ -9,6 +9,7 @@
     <NodeJsExtensionsScriptsTargetFolder>wwwCustomRoot/jay-es</NodeJsExtensionsScriptsTargetFolder>
     <NodeJsExtensionsStylesSourceFolder>NonDefaultScssFolder</NodeJsExtensionsStylesSourceFolder>
     <NodeJsExtensionsStylesTargetFolder>wwwwNonDefaultCssFolder</NodeJsExtensionsStylesTargetFolder>
+    <MarkdownAnalysis>solution</MarkdownAnalysis>
   </PropertyGroup>
 
```

Comment:
@0liver I just noticed, is there any reason for this project to be `<Project Sdk="Microsoft.NET.Sdk.Razor">` instead of `<Project Sdk="Microsoft.NET.Sdk">`?